### PR TITLE
New front page

### PR DESCRIPTION
**Pull request by da-kami**
Wanted to give it a try :)

https://deploy-preview-54--comit-network.netlify.com/

Some more cleanup can be done. 
I will also check the page for typos and missing dots tomorrow.
Feel free to help :)

TODO before merge:

- [x] Check for typos
- [x] Remove test event from community calendar
- [x] Create new community coffee event

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -1,332 +1,520 @@
...
+    <div class="grid grid-cols-2 pt-50 md:pt-13">
+      <div class="pr-40 md:pr-10 my-auto">
+        <img class="float-right w-92 md:w-33 md:pt-20" src="img/case.svg" alt="OPEN DEVELOPER CENTRIC TOOLS">
+      </div>
+      <div class="text-left">
+        <h3>OPEN DEVELOPER <br>CENTRIC TOOLS</h3>
+        <p class="md:pt-4 ">
+          Use <strong>create-comit-app</strong> to spin-up a<br class="hidden md:block"> 
+          development environment. The Javascript SDK<br class="hidden md:block">
+          allows simple usage of COMIT’s powerful API.
```
Comment (bonomat) [inline static/index.html:134]: Do we want this bold here? If so, we should use it consistently, e.g. 

```suggestion
          Use <strong>create-comit-app</strong> to spin-up a<br class="hidden md:block"> 
          development environment. The <strong>Javascript SDK</strong><br class="hidden md:block">
          allows simple usage of COMIT’s powerful API.
```

File: static/index.html [hunk 2]
```diff
@@ -1,332 +1,520 @@
...
+  <!-- /.block-four -->
+  <!-- .block-five -->
+  <div class="px-24 w-full bg-five pb-40 md:pb-0 pt-48 md:pt-32">
+    <h2 class="md:pt-4 mb-4 text-center">COMIT BUILDING BLOCKS</h2>
+    <div class="section m-auto grid grid-cols-11 gap-5 md:gap-3 lg:gap4 pt-20 md:pt-10">
+      <div class="col-span-11 md:col-span-4 flex justify-center md:justify-end">
+        <div class="h-188 w-224 md:h-94 md:w-74 flex flex-col bg-white shadow-theme justify-between items-center border-b-11 md:border-b-5 border-pacific-blue">
+          <img class="w-84 md:w-40 m-auto " src="img/native.svg" alt="Native Atomic Swaps">
+          <div class="text-center mb-4 md:mb-2">
+            <h5>Native (Layer1)<br>Atomic Swaps</h5>
```
Comment (bonomat) [inline static/index.html:249]:
```suggestion
            <h5>Native Atomic <br>Swaps (Layer 1)</h5>
```

File: static/index.html [hunk 3]
```diff
@@ -1,332 +1,520 @@
...
+            <p class="py-10 md:py-4">e.g. Bitcoin To Erc20 Swap</p>
```
Comment (bonomat) [inline static/index.html:250]:
```suggestion
            <p class="py-10 md:py-4">e.g. Bitcoin to Erc20</p>
```

File: static/index.html [hunk 4]
```diff
@@ -1,332 +1,520 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="col-span-11 md:col-span-3 flex justify-center">
+        <div class="h-188 w-224 md:h-94 md:w-74 flex flex-col bg-white shadow-theme justify-between items-center border-b-11 md:border-b-5 border-pacific-blue">
+          <img class="w-94 md:w-40 m-auto" src="img/instant.svg" alt="Instant Swaps">
+          <div class="text-center mb-4 md:mb-2">
+            <h5>Instant Swaps<br>(Layer2)</h5>
```
Comment (bonomat) [inline static/index.html:258]:
```suggestion
            <h5>Instant Atomic<br>Swaps (Layer 2)</h5>
```

**Comment by da-kami** [comment]
> I guess there is no need to review it :D
> Are you taking care of typo-checks?

Yeah, am planning to do a general consistency check (also for dots, ...).
Will most likely do this at some point today or tomorrow.

File: static/index.html [hunk 5]
```diff
@@ -1,332 +1,520 @@
...
+            <p class="py-10 md:py-4">e.g. Ether To Lightning Swap</p>
```
Comment (bonomat) [inline static/index.html:259]:
```suggestion
            <p class="py-10 md:py-4">e.g. Ether to Lightning</p>
```

File: static/index.html [hunk 6]
```diff
@@ -1,332 +1,520 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="col-span-11 md:col-span-4 flex justify-center md:justify-start">
+        <div class="h-188 w-224 md:h-94 md:w-74 flex flex-col bg-white shadow-theme justify-between items-center border-b-11 md:border-b-5 border-pacific-blue">
+          <img class="w-94 md:w-40 m-auto" src="img/preserving.svg" alt="Privacy Preserving Swaps">
+          <div class="text-center mb-4 md:mb-2">
+            <h5>Privacy<br>Preserving Swaps</h5>
+            <p class="py-10 md:py-4">e.g. Bitcoin To Grin Swap</p>
```
Comment (bonomat) [inline static/index.html:268]:
```suggestion
            <h5>Privacy Preserving<br>Atomic Swaps</h5>
            <p class="py-10 md:py-4">e.g. Bitcoin to Grin</p>
```

File: static/index.html (hunk 5 above)
Comment (bonomat) [inline static/index.html:259]:
```suggestion
            <h5>Instant Atomic<br>Swaps (Layer 2)</h5>
            <p class="py-10 md:py-4">e.g. Ether to Lightning</p>
```

File: static/index.html [hunk 7]
```diff
@@ -1,332 +1,520 @@
...
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-    <link rel="stylesheet" href="css/bootstrap.min.css" type="text/css">
-    <link rel="stylesheet" href="css/comit.css" type="text/css">
-    <link rel="shortcut icon" type="image/x-icon" href="img/favicon.ico"/>
-    <link rel="icon" href="images/COMITlogoWbg.png"/>
-    <title>:: COMIT ::</title>
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-RQEDN1PVTD"></script>
```
Comment (da-kami) [inline static/index.html:15]: Check if this has to be readded

File: static/index.html [hunk 8]
```diff
@@ -1,332 +1,520 @@
...
 <html lang="en">
 
 <head>
-    <!-- Required meta tags -->
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-    <link rel="stylesheet" href="css/bootstrap.min.css" type="text/css">
-    <link rel="stylesheet" href="css/comit.css" type="text/css">
-    <link rel="shortcut icon" type="image/x-icon" href="img/favicon.ico"/>
```
Comment (da-kami) [inline static/index.html:11]: Add favicon to new page

File: static/index.html (hunk 3 above)
Comment (bonomat) [inline static/index.html:250]:
```suggestion
            <h5>Native Atomic<br>Swaps (Layer 1)</h5>
            <p class="py-10 md:py-4">e.g. Bitcoin to Erc20</p>
```

**Pull request merged**